test/acceptance/e2e.prepare.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+const util = require('../utils/common');
+
+const files = util.loadFiles(path.join(__dirname, 'e2e'));
```

Comment:
this is missleading, files are not files are directories containing test scenarios right?

Reply:
Files is a list of `path/to/test.js` files.

Reply:
`filepaths` ? 🤔